<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);
QLabel{

 color: rgb(255,255,255);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Heading</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 44, 44);
color: rgb(255, 255, 255);
QLabel{

 color: rgb(255,255,255);

}</string>
    </property>
    <property name="text">
     <string>LISTA DE CADASTRO</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>651</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(141, 179, 129);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>661</width>
       <height>271</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NOME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CURSO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TELEFONE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TOMBO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NOME-LIVRO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NOME-AUTOR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>EMP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DEV</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
